pkg/apis/pxc/v1alpha1/zz_generated.defaults.go

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 package v1alpha1
 
 import (
-	runtime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
I don't think it makes sense since this file is generated by an operator-sdk. https://github.com/percona/percona-xtradb-cluster-operator/blob/master/pkg/apis/pxc/v1alpha1/zz_generated.defaults.go#L19

Reply:
But in the go language, the alias and package name are the same, no need to use an alias

Reply:
yes, but this header is automatically generated by operator SDK.
so on the next run this change become overwrited

Reply:
got it, thanks for your tips.